frontend/tests/register.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { expect, test } from '@playwright/test';
+import { testUser } from './resources/user.data.ts';
+
+test.describe('Register page', () => {
+  test('Register Page exists', async ({ page }) => {
+    await page.goto('http://localhost:3005/');
+    await page.goto('http://localhost:3005/login');
+    await page.goto('http://localhost:3005/register');
```

Comment:
you don't need the top 2 just write 
```suggestion
    await page.goto('/register');
```